### PR TITLE
Few changes in webpack for deployment

### PR DESCRIPTION
**Pull request by boisbb18**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -65,16 +65,18 @@ module.exports = {
     })
   ],
   devServer: {
-    host: 'localhost',
-    port: 8080,
+    host: (process.env.USER === 'root') ? '0.0.0.0' : 'localhost',
```
Comment (Dsalazar1685) [inline webpack.config.js]: We shouldn't be using the dev server at all in the deployed version, so I don't think these changes are necessary...

File: webpack.config.js [hunk 2]
```diff
@@ -65,16 +65,18 @@ module.exports = {
...
+    port: (process.env.USER === 'root') ? 80 : 8080,
     contentBase: BUILD_DIR,
     historyApiFallback: true,
     // respond to 404s with index.html
 
     hot: true,
     // enable HMR on the server
     proxy: {
-      '/api': 'http://localhost:3000',
+      '/api': (process.env.USER === 'root') ? 'http://45.55.247.148:3000' : 'http://localhost:3000',
```
Comment (Dsalazar1685) [inline webpack.config.js]: Same here - the proxy server is only for local use during development.

File: webpack.config.js (hunk 1 above)
Comment (boisbb18) [reply webpack.config.js]: Got it

File: server/server.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const bodyParser = require('body-parser');
 const handler = require('./request-handler.js');
 const PORT = process.env.PORT || 3000;
-
+console.log('POrt --> ', process.env.PORT);
```
Comment (Dsalazar1685) [inline server/server.js]: Plz remove console log

Comment (boisbb18) [reply server/server.js]: Yeah sure thing.

File: .travis/deploy_key.pem [hunk 4]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD1SymHWWk75BntVfeAL1XJCO+glT4SXIqkWCRHsvuYRc++5VdqAOqcGzbzhzL2p2bRe+OML0HmxlQ793cbaiAgJNHusjCP5W1y/woe0+rpLxhWaMmderuqOmLOeh9/lbPWw61FpZRob9tqsn8v3fo1hVOpJets6N1CvGICR2xOZCOlELLakk38mKvifRQWccNF641D2PboGlJ2oNSL5LwKkzYwX1gReZwWIeBvMZpiOSLnWBAf+0SnaAtqUPATeuse8m4Fv6kTyqpWgoJsm1VRnJDDOFOqouyXd6CKtlpabCpQe3mhbiJVLFe10gLawUoL4uSRWiBWlbTlLSw4xOnZ boisbakhodirov@BOISs-MacBook-Pro.local
```
Comment (Dsalazar1685) [inline .travis/deploy_key.pem]: Is this your personal unencrypted ssh key or the encrypted Travis version?

Comment (boisbb18) [reply .travis/deploy_key.pem]: I'm trying to hide. I was testing with travis ci. I'm not sure how secure it should be. I'm gonna put it in .gitignore

**Review by Dsalazar1685 — APPROVED** [review]

**Pull request merged**